docs/conceptual-guides/cookieplone-frontend-addon.md

```diff
@@ -0,0 +1,204 @@
+---
+myst:
+  html_meta:
+    "description": "An explanation on the anatomy of a frontend add-on"
+    "property=og:description": "An explanation on the anatomy of a frontend add-on"
+    "property=og:title": "Frontend add-on anatomy"
+    "keywords": "Seven, cookieplone, template, add-on, anatomy"
+---
+
+# Frontend add-on anatomy
+
+After you create a frontend add-on with the {term}`cookieplone` generator {doc}`../get-started/create-package`, the code lives in `packages/<add-on-name>`.
```

Comment:
This should be an introduction. I moved the original sentence into the next section, and replaced it with an introduction.

```suggestion
This chapter explains the anatomy of a frontend add-on in Seven that was created with {term}`cookieplone`.
```